src/js/themes/hpe.js

```diff
@@ -1346,6 +1351,9 @@ export const hpe = deepFreeze({
       ],
     },
   },
+  starRating: {
+    color: 'green',
```

Comment:
What is the rationale behind having different colors for star ratings vs. thumb ratings? 

I believe the Figma component has the color `purple!` for both.

Reply:
I can change to purple in Megans originally feedback she had green for stars

Reply:
Understood. I'd just like us to be intentional in our design choices. In this case green in one spot and purple in another seems arbitrary unless we can explain why. If not, I'd advocate for consistency, plus align with the Figma component.

Reply:
agree ☝️